code/datums/tutorial/xenomorph/abomination.dm

```diff
@@ -0,0 +1,214 @@
+/datum/tutorial/xenomorph/abomination
+	name = "Xenomorph - Abomination"
+	desc = "A tutorial to teach you how to play the Abomination xenomorph caste. Completing this is required to be able to play an Abomination."
+	icon_state = "predalien"
+	tutorial_id = "xeno_abom_1"
+	tutorial_template = /datum/map_template/tutorial/s7x7
+	starting_xenomorph_type = /mob/living/carbon/xenomorph/predalien/tutorial
+
+// START OF SCRITPING
+
+/datum/tutorial/xenomorph/abomination/start_tutorial(mob/starting_mob)
+	. = ..()
+	if(!.)
+		return
+
+	init_mob()
+	xeno.melee_damage_lower = 40
+	xeno.melee_damage_upper = 40
+	xeno.lock_evolve = TRUE
+
+	message_to_player("Welcome to the tutorial for the Abomination xenomorph. As an Abomination, you are a frontline powerhouse whose damage scales with your kill count.")
+	message_to_player("Your kill count scales when you kill humans with your slash attack, up to 10 kills. Ability kills (including Tail Stab) do not count towards this.")
+
+	addtimer(CALLBACK(src, PROC_REF(how_to_be_abom)), 10.5 SECONDS)
```

Comment:
Basically all steps progress too quickly to ever manage to read what appears on screen. They can read it in chat and it eventually gets to a point where it is waiting on the player to do something, but it is a ton of information getting dumped very quickly.

Reply:
Even though I think it's readable (I read fast, so I might be the wrong person to ask), I'll noodle on this for a bit